packages/git-extractor/src/routes/github/pull/download.ts

```diff
@@ -48,6 +47,9 @@ export async function downloadRepository(
 
   const result: INormalizedModules = {};
 
+  const pathArray = <string[]>[];
```

Comment:
**suggestion:** It looks like we can use type assertions here rather than typecasting.

```suggestion
  const pathArray: string[] = [];
```

Reply:
Ohhh I had no idea! I just looked up "how to type array in typescript" haha. Thanks for the clarification!